packages/reakit/src/Popover/__tests__/Popover-test.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from "react";
+import { render } from "react-testing-library";
+import Popover from "../Popover";
+
+test("html attrs", () => {
+  const { getByText } = render(
+    <Popover id="test" aria-label="test">
+      Test
+    </Popover>
+  );
+  expect(getByText("Test")).toHaveAttribute("id", "test");
+  expect(getByText("Test")).toHaveAttribute("aria-label", "test");
+});
+
+test("placement", () => {
+  const { container } = render(<Popover placement="right" />);
+  expect(container.firstChild).toHaveAttribute("data-placement", "right");
+});
+
+// test("flip", () => {
+//   const { container } = render(<Popover flip={false} />);
+//   expect(container.firstChild).toHaveAttribute("data-placement", "right");
+// });
+
+// test("shift", () => {
```

Comment:
For the "shift" and "flip" props I'm not actually seeing any functionality in Codesandbox - should these be doing something visual @diegohaz @PlayMa256?

Reply:
One of them is working, i just don't know, based on the `popper.js` docs, i would say that only shift is reproducible on the playground. It is when you hit the popover with the boundary of your browser, it should change directions. I actually don't know how you could test it...

Reply:
Ok I'll dive into Popper.js repo when I get some time and see if I can dig out how they test something like this

Reply:
After a bit of research I'm not sure it's particularly trivial to test this so maybe for another time/separate issue?

Reply:
@diegohaz Removed them in my last commit so should be good to go!